### DISCUSSION TITLE
How do you drop vertexes?

### DISCUSSION THREAD

**Discussion by ZachMurray** [ArcadeData/arcadedb#1094] [Q&A] [answered]
Hi ArcadeDB Discussions! 

How do you drop all the vertexes from an ArcadeDB? 
Looking for something Gremlin like `g.V().drop()`. 
There seems like a way to be able to do this with `txBlock`s but I'm clueless on how to initialize one. 
```java
        remoteDatabase.begin();
        BasicDatabase.TransactionScope txBlock = null;
        remoteDatabase.transaction(txBlock);
        remoteDatabase.drop();
        remoteDatabase.create();
        remoteDatabase.commit();
```

**Comment by lvca** [comment, marked as answer]
Are you looking to drop only vertices, or also edges? Or rather the entire database?

About the transaction, you could use a lambda function like this:

```java
remoteDatabase.transaction( ()->{
  remoteDatabase.command( "gremlin", "g.V().drop" );
} );
```

Or if you want to drop the entire database from a remote database API, you could do:


```java
remoteDatabase.drop();
```

**Reply by ZachMurray** [reply]
Thanks Ivca!
